ios/Plugin/PurchasesPlugin.swift

```diff
@@ -138,12 +149,12 @@ public class PurchasesPlugin: CAPPlugin, PurchasesDelegate {
             call.reject("aPackage parameter did not have identifier key")
             return
         }
-        guard let offeringIdentifier = package["offeringIdentifier"] as? String else {
-            call.reject("aPackage parameter did not have offeringIdentifier key")
+        guard let presentedOfferingContext = package["presentedOfferingContext"] as? [String: Any] else {
+            call.reject("aPackage parameter did not have presentedOfferingContext key")
             return
         }
         CommonFunctionality.purchase(package: packageId,
```

Comment:
Yeah yeah! I briefly and casually talked to Nacho about this a while back because I realized that Android allows it for products but iOS does not.

I _do_ think this should be done but maybe we can do this in a follow up PR? It would require a Hybrid Common changes to do this and it would need to be updated on all the SDKs 😬 I'm happy to do this followup PR though!